### DISCUSSION TITLE
docs/sweetpotato-carrot-mash-with-garlic-and-pecans/

### DISCUSSION THREAD

**Discussion by giscus** [Yarden-zamir/me#24] [Announcements]
# docs/sweetpotato-carrot-mash-with-garlic-and-pecans/

https://www.yarden-zamir.com/docs/sweetpotato-carrot-mash-with-garlic-and-pecans/

**Comment by Yarden-zamir** [comment]
![pic](https://lh3.googleusercontent.com/pw/ABLVV84cNRq6t7bm2fmily9HK1Hfe9yjXmpaT1jMj4hfPEgmQqtBVVkS837gwczLaqcoghzIpij5BBuIrCGTri9Xot1fUF2812Xg1r7jeBHfJNu8eY2xIs2Kd86QfJM2yqffirKd_uJkVnQd4zY2U9ZEVNjSNsJt921QATyRt0G69A5sg5NHTK8QSX4IY9kBwe1wAzkXTKupHqSFFyAvRoJ__ie3WF7UqOSrGem9aaCkfotX1LNRqkibSTt37HKcprukEFNgvoCXskBjQdHscNVCounN9L-UtSrN6OY355sGXRkjDI4bBiQRK0GgTcncR-qGo-VOrPAL8nXRmAuF2-9PDP3gubo_i33zAWKIwi35FW_lY-n7i4vXDOCWAlagebXhpe-BcIJ4z8t0sfgR_ljx90Gq0DBDGWA2-RWT-lli7vc-u-LgXXBBp7msSq_sLvtMXIUT0UAQPBGdpUoUTW5h8W0aasQ49Vvj15ik2LEU7cnmBmccuhoWZiCwtSR_ozDSjZhYyVL01UK7AVnZP2ItHi1zryqd4gcWki-DxgzPTU_PN3pXk-5IL1rSiFBXrDyzZJGHtt3qH0zqjXktcFNjp760Sha1vA9_5GkExll_1mLI9852IWCNutUiwEbNXvt0jrPOjXjlPAtOLdb1X0-bunjD8wsu0LmPWqMK9s5EdVhpM1MdCc0R2YtM3pMRGGyiEJ8Yi5RPQv-YExT2bNIQLboVfHUPdLlMORiqM7CqxEfMyjVAKott7j6AncUgQCia9u9pgNQ9i5BC71UzP_2Pbor2xzgqGVDaInKJfDydqIYKKHs6-aRbChHfX_WzGK4UaTJmHsVLpiEqVWHH4CHwICNJsPRfCnHKKPyjQt3gE9fl99EcIxSFYsu_3XfWamNe45hYEzrz0LSTR4Eq5s_bK66gK0a9fiCF2Mr5dHh1W3_Jg7I82HodJflat-OWljqXQXlNpVog-oPSjOwtad50JPM-j3PQavo5CyM=w910-h910-s-no-gm?authuser=0)

**Comment by Yarden-zamir** [comment]
![](https://lh3.googleusercontent.com/pw/ABLVV84DxB2oMHmUAbkLUMxhNRK7nWzfoqzjCmJg-dQTSE_ewYUPhqBof5pXm_SOkFlK8EgqxHeBAqvgVe6Mxz-lBGgZPdYIAshMsp6bugcT2KNIrm8odBRN8K5tFrWszMPK2AbLVLT_H_vkZ-w5SiAXANf3_O5rlR0u9YD5hYfmEzVtuwtijXzo6q6YEikJkkk-f6KK3Zcpdto8KaD5fx9atGZ-x0rj0NsYnkruXL71l5W71elFjFDD_kLaWKIPPBTsnrDxYA2ihY6K4alXWnLc0LJdWVkJC5cjLhk7MwHeo3lsVRXUuG1uQZEz7Hk36Vp1Yh8flasTLo9rQ2KgC5mXmjMHtwgbIwpU7-qvFuAYifEw01MkH9uxtVgJgKXXZUW3lijuWhM22uZJ7BYx0jBvmxrYZbqIbf0sNcJAG52exlvrx46XTTh3F6UP8Yvwd2-zf8pSrPz6NB8uWErseD6AYTxEAe-p2J4WzNLwvBlgfb3SrOBFdxbKe3BMahA-DS0TswycjJQhs_Z-ekEWa6LRnGpoGexwOi4Q7v0BhotJAMl42XI-0U2_fz7CmJLaiGhQxlT-9h2_x4-CvacBz4_GlAmSKPUDhpyU28h-ZRQcRP82IaN265EzZGYxh3QUTpLS7wMxh3j-0K7FVwHW5k-CS49zJRLP9VTWzBykdY1L2visM1owa5dj5FcD-XZSW0g12YqdxzO3T2JXwacG4aZ39M1dMvz6pO8Ob_GDVpvNu3pXsx90AlRkRoLpsfEsW44XNxGIoN7aG2fAnMQWyuw-eGOC8x59Wt0xuL10GQmw486wvp3tzOHq3G-44H8APU2ppFXfBYM31CAZnyrswws5oTvqN-rWTRuS6Q0UGW4dpWnTCcyuSNd49mv-eYLZDDCtKonAmxbjzsrhkUZllgcZMouWCLrFdz0MeG-qjea3czW_POaM5FmaCT-x1QNTBtSXVwKu7sUsV-WEyB4nF48kucCe_z03i2hxES8=w410-h910-s-no-gm?authuser=0)
